cg/cli/compress/fastq.py

```diff
@@ -186,3 +188,59 @@ def decompress_spring(context, case_id, dry_run):
         return
 
     LOG.info("Decompressed spring archives in %s individuals", decompressed_inds)
+
+
+@click.command("spring-flowcell")
+@click.argument("flowcell_id")
+@click.option("-d", "--dry-run", is_flag=True)
+@click.pass_context
+def decompress_spring_flowcell(context, flowcell_id, dry_run):
+    """Decompress SPRING file, and include links to FASTQ files in housekeeper"""
+    LOG.info("Running decompress spring")
+    compress_api = context.obj["compress_api"]
+    update_compress_api(compress_api, dry_run=dry_run)
+
+    store = context.obj["status_db"]
+    samples = store.get_samples_from_flowcell(flowcell_id=flowcell_id)
+
+    decompressed_inds = 0
+    try:
+        for sample_id in samples:
+            was_decompressed = compress_api.decompress_spring(sample_id)
+            if was_decompressed is False:
+                LOG.info("skipping individual %s", sample_id)
+                continue
+            decompressed_inds += 1
+    except CaseNotFoundError:
+        return
+
+    LOG.info(f"Decompressed spring archives in {decompressed_inds} individuals")
```

Comment:
As someone that does this in production this would be more logical if it said "Started.... individuals" since it actually does not decompress on the spot